src/steamship/agents/examples/chatgpt_api.py

```diff
@@ -0,0 +1,22 @@
+"""Minimum viable AgentService implementation.
+
+This will result in an agent that effectively acts like ChatGPT.
+"""
+
+from steamship.agents.functional import FunctionsBasedAgent
+from steamship.agents.llms.openai import ChatOpenAI
+from steamship.agents.service.agent_service import AgentService
+from steamship.utils.repl import AgentREPL
+
+
+class MyAssistant(AgentService):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self._agent = FunctionsBasedAgent(llm=ChatOpenAI(self.client), tools=[])
+
+
+if __name__ == "__main__":
+    AgentREPL(
+        MyAssistant,
+        agent_package_config={"botToken": "not-a-real-token-for-local-testing"},
```

Comment:
do you need `botToken` config here ?

Reply:
Argh. Sloppy of me. Thanks for the catch!